release notes/v0.2.0.md

```diff
@@ -0,0 +1,122 @@
+This release adds three new features:
+* HTTP Basic Auth support.
+* Automatically adjusting viewport insets depending on the operating system.
+* Denying IPs and hostnames.
+
+Fixes bugs, including a major one in `WaitForFrameNavigation,` and we now handle the extension shutting-down stage more robustly. Improves testability, logging, and user error messages.
```

Comment:
I wouldn't want to highlight the `WaitForFrameNavigation` fix; it's only relevant for dynamic sites, and we want them to be using `waitForSelector()` anyway.

```suggestion
In addition, it comes with a few important bug fixes, and we now handle the extension shutting-down stage more robustly. It also improves testability, logging, and user error messages.
```